internal/nodes/dequeue_ingestion.go

```diff
@@ -40,6 +40,11 @@ func DequeueIngestion(key string, fullsync bool) {
 	sharedQueue := utils.SharedWorkQueue().GetQueueByName(utils.GraphLayer)
 
 	objType, namespace, name := lib.ExtractTypeNameNamespace(key)
+
+	if objType == lib.L7Rule {
```

Comment:
This logic will be hit only from `FullSyncK8s`. As we are not doing anything here, just returning. Can we take out going through L7Rule logic from FullSyncK8s (Keep validation only don't enqueue the key).

Reply:
Lets say L7Rule changed from valid to invalid or invalid to valid during AKO bootup. How are we handling that transition like applying L7Rule or taking out L7Rule from VS fields.

Reply:
removed the enqueue logic from FullSyncK8s. This transition will be handled when hostrule will be processed.
L7 rule is validated before hostrule so latest ststus of L7 rule will be used in hostrule processing.